### PR TITLE
fix: fix sorting of results and build docker image example

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: multires-lyrics-search/tests/query_results.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
...
+        "Well they left then in the morning, a hundred pairs of wings. In the light moved together in the colors of the morning. I looked to the clouds in the cirrus sky and they'd gone.. Across the marshes, across the fields below.. I fell through the vines and I hoped they would catch me below.. If only to take me with them there,. Tell me the part that shinesIn your heart on the wind.. And the reeds blew in the morning.. Take me along to the places. You've gone when my eyes looked away.. Tell me the song that you sing in the trees in the dawning.. Tell me the part that shines in your heart. And the rays of love forever,. Please take me there..",
```
Comment (cristianmtr) [inline multires-lyrics-search/tests/query_results.json:6]: Make sure these are in the same order as the chunks. The chunks seem to be in reverse (best first)

**Review by cristianmtr — CHANGES_REQUESTED** [review]
Fix order

**Review by nan-wang — DISMISSED** [review]
LGTM👍

File: multires-lyrics-search/README.md [hunk 2]
```diff
@@ -52,36 +52,34 @@ pip install -r requirements.txt
...
+
+```bash
+docker run -p 65481:65481 jinahub/app.example.multireslyricssearch:0.0.1-0.9.17
+```
 
 
 ## License
 
-Copyright (c) 2020 Han Xiao. All rights reserved.
+Copyright (c) 2020-2021 Han Xiao. All rights reserved.
```
Comment (cristianmtr) [inline multires-lyrics-search/README.md:83]: Copyright should be Jina AI, no?

Comment (JoanFM) [reply multires-lyrics-search/README.md:83]: not sure about these things

File: multires-lyrics-search/Dockerfile [hunk 3]
```diff
@@ -1,11 +1,32 @@
-FROM jinaai/jina:0.4.1-devel
+FROM jinaai/jina
 
-ADD requirements.txt .
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
 
-RUN apt-get update && pip install -r requirements.txt
+RUN pip uninstall -y jina && pip install -r requirements.txt && mkdir data && mv lyrics-data.csv data/
```
Comment (cristianmtr) [inline multires-lyrics-search/Dockerfile:7]: Why are you removing the command for the data? I am getting 

```
mv: cannot stat 'lyrics-data.csv': No such file or directory
The command '/bin/sh -c pip uninstall -y jina && pip install -r requirements.txt && mkdir data && mv lyrics-data.csv data/' returned a non-zero code: 1
```

Comment (cristianmtr) [reply multires-lyrics-search/Dockerfile:7]: Ah no sorry, that's another problem

Comment (cristianmtr) [reply multires-lyrics-search/Dockerfile:7]: It seems this `lyrics-data.csv` is missing.

File: multires-lyrics-search/Dockerfile [hunk 4]
```diff
@@ -1,11 +1,32 @@
...
+
+ENTRYPOINT ["python", "app.py", "search"]
+
+LABEL author="Jina AI Dev-Team (dev-team@jina.ai)"
+LABEL type="app"
+LABEL kind="example"
+LABEL avatar="None"
+LABEL description="Jina App using recursive Document structure to search for lyrics from a songs database, the database contains 10k songs"
+LABEL documentation="https://github.com/jina-ai/examples/tree/master/multires-lyrics-search"
+LABEL keywords="[examples, faiss, annoy, ANN]"
```
Comment (cristianmtr) [inline multires-lyrics-search/Dockerfile:25]: minor thing: keywords are wrong

Comment (JoanFM) [reply multires-lyrics-search/Dockerfile:25]: ohh, can u provide q suggestion to the keywords so I can merge here. for next time we rebuild it it should be good

**Review by cristianmtr — APPROVED** [review]

**Pull request merged**